coderd/authorize_test.go

```diff
@@ -29,12 +33,27 @@ func TestCheckPermissions(t *testing.T) {
 	orgAdminUser, err := orgAdminClient.User(ctx, codersdk.Me)
 	require.NoError(t, err)
 
+	version := coderdtest.CreateTemplateVersion(t, adminClient, adminUser.OrganizationID, nil)
+	coderdtest.AwaitTemplateVersionJob(t, adminClient, version.ID)
+	template := coderdtest.CreateTemplate(t, adminClient, adminUser.OrganizationID, version.ID)
+
+	// This unit test needs the enterprise version of coder to update the ACL
+	// list. This is just to setup the object to make sure the ACL lists are
+	// honored when checking a single resource.
+	// Because the enterprise api is not available, I use the database
+	// directly.
+	err = api.Database.UpdateTemplateUserACLByID(ctx, template.ID, database.TemplateACL{
+		memberUser.ID.String(): []rbac.Action{rbac.WildcardSymbol},
+	})
+	require.NoError(t, err)
```

Comment:
I need the enterprise route to setup the test :/. But I don't have enterprise routes enabled in this test :thinking:

Reply:
This same test exists for Enterprise, just do it there!

Reply:
I can copy the test in both places, and only have this test case in enterprise.

The downside is I don't get line coverage across packages. So that's why I want to also keep this test here